### PR TITLE
DF 2267 - Update Event macros to display content based on state.

### PR DESCRIPTION
**Pull request by sebworks**
Updated event macros to use Sheer 'when' function in order to display content based on state.

## Changes
- Updated src/events/_event-post-macros.html to use Sheer 'when' function.
- Updated 'src/events/_event.html' to use Sheer 'when' function and display appropriate event_status content.
-  Updated 'src/events/_single.html' to use Sheer 'when' function and display appropriate event_status content. 
- Removed 'src/events/archive/_single.html' because all events will use the same `_single.html` file.

## Testing

Future Event
- Visit 'http://localhost:7000/events/fall-consumer-advisory-board-meeting-in-washington-dc/'

Archived Event
- Visit 'http://localhost:7000/events/summer-consumer-advisory-board-meeting-in-omaha-ne/'

## Screenshots
screen shot 2015-07-24 at 4 53 06 pm

screen shot 2015-07-24 at 4 54 04 pm

## Notes 
- This PR doesn't address the video playing capability while an event is in progress. I will address that in a separate PR.
- Currently using placeholder image for archived events. Will need to determine what properties to use to display the images.

### CODE REVIEW

File: src/events/_event.html [hunk 1]
```diff
@@ -61,39 +64,39 @@ <h1 class="post_heading">
...
+        {% if (event_state != 'past') %}
+        <aside class="post_inset post_inset__right line-container event-status">
+            <div class="line-container_body">
+                <h1 class="u-visually-hidden">Event reservation and viewing details</h1>
+                <div class="event-status_rsvp">
+                  <p>
+                      <strong>This event requires an RSVP.</strong>
+                  </p>
+                  <p>
+                      <a href="mailto:reserve@cfpb.gov?subject=Event RSVP&body=To RSVP, please fill in your first and last name below and send this email.%0D%0A%0D%0AFirst name:%0D%0ALast name:" class="btn"><span class="btn_icon__left cf-icon cf-icon-email-social"></span> reserve@cfpb.gov</a>
```
Comment (anselmbradford) [inline src/events/_event.html]: Maybe break this across a few lines.

File: src/events/_event.html [hunk 2]
```diff
@@ -61,39 +64,39 @@ <h1 class="post_heading">
...
+                  </p>
+                </div>
+                {% if (event.live_stream) %}
+                <div class="event-status_livestream">
+                  <p>
+                      {# TODO Change to livestream icon when that's added to cf-icons. #}
```
Comment (anselmbradford) [inline src/events/_event.html]: livestream icon is in cf-icons now https://github.com/cfpb/cf-icons/blob/gh-pages/demo/index.html#L703

File: CHANGELOG.md [hunk 3]
```diff
@@ -48,12 +48,15 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
...
   `sub-page_content`, and `sub-page_content-markup` class areas.
+- Updated event macros to use Sheer 'when' function in order to
+  display content based on state.
 
 ### Removed
 - Removed requestAnimationFrame polyfill.
 - Removed `_tests/browser_tests/README.md`, `_tests/macro_testing/README.md`, `_tests/processor_tests/README.md`.
 - Removed `grunt vendor` from `setup.sh`.
 - Removed unused CSS on `office.less`
+- Removed `/events/archive/_single.html`
```
Comment (jimmynotjim) [inline CHANGELOG.md]: Good catch, meant to do this when I originally tied in the data.

File: src/events/_event.html (hunk 2 above)
Comment (anselmbradford) [inline src/events/_event.html]: L:85 just needs to be removed. It's good on L:86.

File: src/events/_event.html [hunk 4]
```diff
@@ -61,39 +64,39 @@ <h1 class="post_heading">
...
-                          <span class="event-meta_description event-meta_description__block">
-                              Video link available:
-                          </span>
-                          {% from "_event-post-macros.html" import event_meta_datetime as event_meta_datetime %}
-                          {{ event_meta_datetime(event.live_stream.date) }}
-                      </p>
-                    </div>
-                    {% endif %}
+        {% if (event_state != 'past') %}
+        <aside class="post_inset post_inset__right line-container event-status">
```
Comment (jimmynotjim) [inline src/events/_event.html]: We should explore this line-container and it's body. The styles that are added seem to be doable with other existing code.

**Comment by anselmbradford** [comment]
:+1: LGTM

**Comment by jimmynotjim** [comment]
:+1: from me after you address @anselmbradford comments

**Pull request merged**